http/servlet-undertow/src/test/java/io/quarkus/ts/http/undertow/UndertowMissServletClassIT.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.ts.http.undertow;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.scenarios.annotations.DisabledOnQuarkusVersion;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+@DisabledOnQuarkusVersion(version = "3\\.(8|9|10|11|12|13|14)\\..*|3\\.15\\.(0|1)(\\..*)?", reason = "Disabled on Quarkus versions before 3.15.2 due to build fail because missing servlet class configuration undertow issue https://github.com/quarkusio/quarkus/issues/43825")
```

Comment:
Looking at this and I think it should not been here.

When test it with 3.15.1 build fail even with this. As it affect whole module. 

I think you have 2 options here:

1. Implement the option for custom file into the FW. So you can add/overwrite files
2. You try to replace it in `app.onPreStart()`
3. Maybe there is another option

Reply:
I've pushed new changes with a possible another option such `onPreStart` .

Reply:
Apologies, working on some proper solution